lib/polyfill/media_capabilities.js

```diff
@@ -217,6 +235,60 @@ shaka.polyfill.MediaCapabilities = class {
     return res;
   }
 
+  /**
+   * Checks if the given media parameters of the video or audio streams are
+   * supported by the Cast platform.
+   * @param {{
+   *   contentType: string,
+   *   width: (number|undefined),
+   *   height: (number|undefined),
+   *   frameRate: (number|undefined),
+   *   transferFunction: (string|undefined)
+   * }} options canCastDisplayType() options.
+   *     contentType: A valid MIME type and (optionally) a codecs parameter.
+   *     width: Describes the stream horizontal resolution in pixels.
+   *     height: Describes the stream vertical resolution in pixels.
+   *     frameRate: Describes the frame rate of the stream.
+   *     transferFunction: Describes the video transfer function supported by
+   *         the rendering capabilities of the user agent.
```

Comment:
Since all of these come from videoConfig, I feel like you should just pass that instead of creating a new record type.

Reply:
Done, thanks